tests/conftest.py

```diff
@@ -1730,17 +1747,27 @@ def finalizer():
         request.addfinalizer(finalizer)
 
 
+@pytest.fixture()
+def rgw_bucket_factory(request, rgw_obj):
+    return bucket_factory_fixture(request, mcg_obj=None, rgw_obj=rgw_obj)
+
+
+@pytest.fixture(scope='session')
+def rgw_bucket_factory_session(request, rgw_obj_session):
+    return bucket_factory_fixture(request, mcg_obj=None, rgw_obj=rgw_obj_session)
+
+
 @pytest.fixture()
 def bucket_factory(request, mcg_obj):
```

Comment:
not a must but i would change this factory to be called mcg_bucket_factory same as the rgw

Reply:
This would be more correct from a documentation and readability standpoint, but would seriously and heavily affect and break backwards compatibility with existing code. Thus, I opted to not change its name. 
I'll add a comment regarding this.